<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serviceman Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/soldering_iron.png</normaloff>:/icons/soldering_iron.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="idInputField">
        <property name="placeholderText">
         <string>Wpisz SN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Szukaj</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="deviceListView">
        <property name="maximumSize">
         <size>
          <width>275</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="repairsTableView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClients"/>
   <addaction name="actionModels"/>
   <addaction name="actionDamages"/>
   <addaction name="actionTechnicians"/>
   <addaction name="separator"/>
   <addaction name="actionDatabaseSettings"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionClients">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/clients.png</normaloff>:/icons/clients.png</iconset>
   </property>
   <property name="text">
    <string>Klienci</string>
   </property>
  </action>
  <action name="actionModels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/devices.png</normaloff>:/icons/devices.png</iconset>
   </property>
   <property name="text">
    <string>Modele</string>
   </property>
   <property name="toolTip">
    <string>Modele</string>
   </property>
  </action>
  <action name="actionDamages">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/damages.png</normaloff>:/icons/damages.png</iconset>
   </property>
   <property name="text">
    <string>Usterki</string>
   </property>
  </action>
  <action name="actionTechnicians">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/technicians.png</normaloff>:/icons/technicians.png</iconset>
   </property>
   <property name="text">
    <string>Serwisanci</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </action>
  <action name="actionDatabaseSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
   </property>
   <property name="text">
    <string>Ustawienia bazy danych</string>
   </property>
   <property name="toolTip">
    <string>Ustawienia bazy danych</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
